### PR TITLE
Feature/us012 request forgotten password

### PR DESCRIPTION
**Pull request by duncannixon**
Fixes a sign-in problem on master. Also includes some initial screen work on forgot / reset password  - doesn't break anything so safe to merge to master. Also includes changes to the dev-only home page to aid handover and demos.

### CODE REVIEW

File: app/templates/passwords/reset-password.html [hunk 1]
```diff
@@ -50,24 +55,26 @@ <h1 class="saturn">Reset your password</h1>
     </ul>
 
     <fieldset id="reset-details" class="js-new-password-group">
-        {% if errorType != None %}
+
+        {% if errorType.password %}
```
Comment (andrewmil) [inline app/templates/passwords/reset-password.html]: The password field is displaying an error with javascript code when the password complexity is not met. There is an identical problem on the register-account-details page so could fix there too

File: app/views/passwords.py [hunk 2]
```diff
@@ -1,45 +1,72 @@
...
+        # TODO do some kind of back end processing to validate the email address, error handling, logging
+        # print('Email address=' + email_address)
+
+        return redirect(url_for('passwords_bp.forgot_password_check_email'))
+
     template_data = {
         "error": {
-            "type": request.args.get("error")
+            "type": form.errors,
+            "logged_in": "False"
```
Comment (andrewmil) [inline app/views/passwords.py]: is logged_in used anywhere for this?

Comment (duncannixon) [inline app/views/passwords.py]: Thanks - logged_in is not used so I have taken it out.

File: app/templates/passwords/reset-password.html (hunk 1 above)
Comment (duncannixon) [inline app/templates/passwords/reset-password.html]: Agreed, suggest confirming the requirements with the RAD team and either removing the javascript as a defect or meet their defined requirement as an iteration.

**Pull request merged**